Docker.DotNet.BasicAuth.nuspec

```diff
@@ -0,0 +1,31 @@
+﻿<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>Docker.DotNet.BasicAuth</id>
+    <version>2.124.3</version>
```

Comment:
I was trying to just get the nuget metadata from the `project.json` to a .nuspec the way it was in the repo. But yes, will look on how to do that with appVeyor once I have everything else approved.